### PR TITLE
remove yaml in manifest.in

### PR DESCRIPTION
**Pull request by xiangyan99**
*.yaml under tests folder are recorded tests. Should not ship them in sdist. Remove *.yaml from manifest.in

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/cognitiveservices/azure-cognitiveservices-personalizer/README.rst [hunk 1]
```diff
@@ -3,30 +3,19 @@ Microsoft Azure SDK for Python
 
 This is the Microsoft Azure Personalizer Client Library.
 
-This package has been tested with Python 2.7, 3.4, 3.5, 3.6 and 3.7.
+This package has been tested with Python 2.7, 3.5, 3.6 and 3.7.
```
Comment (lmazuel) [inline sdk/cognitiveservices/azure-cognitiveservices-personalizer/README.rst]: Seems personalizer needs to have "auto_update=false" like most of data-plane. I suggest you create a commit with that, and you "rebase --interactive" to remove the "personalizer" commit, and then you "push --force"

File: tools/azure-sdk-tools/packaging_tools/templates/MANIFEST.in [hunk 2]
```diff
@@ -1,4 +1,4 @@
-recursive-include tests *.py *.yaml
+recursive-include tests *.py
```
Comment (lmazuel) [inline tools/azure-sdk-tools/packaging_tools/templates/MANIFEST.in]: You don't own this file, that's a discussion to have outside of this PR with Zim.

File: sdk/cognitiveservices/azure-cognitiveservices-personalizer/README.rst (hunk 1 above)
Comment (lmazuel) [reply sdk/cognitiveservices/azure-cognitiveservices-personalizer/README.rst]: Or just remove the commit :)

**Pull request closed without merging**